index.html

```diff
@@ -179,6 +231,7 @@ <h2 class="about-me-h2">About me</h2>
       </ul>
       <img src="img/blackbar.png" alt="" class="black-bar">
     </div>
```

Comment:
-- [ ] Kindly add a little bit of padding space between the social icons to perfect your work and fit the design given 👍 . Good job

## Design:
![image](https://github.com/bohaz/First-mobile-project/assets/50990925/af32f1d2-771b-4425-af3a-b3ad6beccd3d)

## Figma:
![image](https://github.com/bohaz/First-mobile-project/assets/50990925/9af1112e-1214-4661-af5c-094dc14a8cc9)